mysql-test/suite/versioning/t/vtmd.test

```diff
@@ -36,7 +36,7 @@ create or replace procedure check_vtmd (in vtmd_name varchar(64))
 begin
   set @tmp= concat('
     create or replace temporary table
-    tmp_vtmd as
+    tmp_vtmd with system versioning as
```

Comment:
Without such syntax extension it would be impossible to create system versioned table because it requires `SYSTEM_TIME`.
> A table definition defines either a regular persistent base table, a system-versioned table, a global temporary table, or a created local temporary table.
Case:
a) If GLOBAL is specified, then a global temporary table is defined.
b) If LOCAL is specified, then a created local temporary table is defined.
c) If WITH system versioning clause is specified and a table period definition that specifies SYSTEM_TIME is specified, then a system-versioned table is defined.
d) Otherwise, a regular persistent base table is defined.

So, probably it shouldn't be possible to create system versioned tables from `SELECT` at all. I suggest to discuss it with MariaDB guys.